### ISSUE TITLE
How to connect & get data from sensor FWM8BLZ02

### ISSUE THREAD

**Issue by TokyoTimo** [plathome/docs#2]
Can you please also send me information about sensors:

Model:FWM8BLZ02

How to connect them?
How to get data from them?
What kind of the sensor is it?

**Comment by daizoom** [comment]
> How to connect them?
> How to get data from them?
Could you please check "Chapter 5 Items Related to Bluetooth Device" on "Web UI Setup Guide" and "Data Acquisition Guide"?

> What kind of the sensor is it?
FWM8BLZ02 has temperature and accelerometer sensor.

**Comment by Vladyslav93** [comment]
The FWM8BLZ02-109042 model is Bluetooth smart beacon without sensors, and the FWM8BLZ02A-109047 model is a Bluetooth Smart sensor beacon with temperature and accelerometer sensors.  I think @daizoom that your sensor has not sensors.

**Comment by daizoom** [comment]
@Vladyslav93 We support both devices.
Could you check "2-1-3. Device Information Transmission Setting" on "Data Acquisition Guide"?

**Comment by Vladyslav93** [comment]
@daizoom  Yes, I understood that it possible to use both, but if I realized correctly information from the link: https://www.acalbfi.com/nl/IoT-and-Wireless/Bluetooth/p/Bluetooth-Smart-Beacon-with-Sensors/0000008N0S
There are 2 types of beacon FWM8BLZ02 this is a just smart beacon and FWM8BLZ02**A** this is the beacon with the sensor. You should check which type you have.

**Comment by daizoom** [comment]
@Vladyslav93 I have both types of devices and I guessed you have the beacon with sensors.
Therefore I wrote
> Could you check "2-1-3. Device Information Transmission Setting" on "Data Acquisition Guide"?

If you have the beacon without sensor, could you refer to "2-1-2. Beacon Transmission Setting" on "Data Acquisition Guide"?

**Comment by Vladyslav93** [comment]
@daizoom Sorry but I can not find this destination setting!!! On the manual, I can not find information on which tab should I do it.

**Comment by Vladyslav93** [comment]
@daizoom How to work with beacons, if I need to disable "Data Acquisition" to have the possibility to connect to the beacon, and from another side, I need to enable it to work with beacon sensor data???? Could you please help?

**Comment by daizoom** [comment]
@Vladyslav93 In case to detect ble beacon on Web UI, you need to disable "Data Acquisition", but when getting data, you need to enable "Data Acquisition".
